src/transformers/configuration_xlnet.py

```diff
@@ -110,6 +110,8 @@ class XLNetConfig(PretrainedConfig):
                 Used in the SQuAD evaluation script for XLM and XLNet.
             end_n_top (:obj:`int`, optional, defaults to 5):
                 Used in the SQuAD evaluation script for XLM and XLNet.
+            use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
+                Differs slightly from other models as it is always turned on at training time.
```

Comment:
I would put the base docstring here and add a note for XLNet specific behavior:
```
use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
    Whether or not the model should return the last key/values attentions (not used by all models).
    
    .. note::
        This flag behaves differently as with other models: it just controls the inference behavior and is always turned on at training time.
```
while adapting the first sentence if needed.

Reply:
Oh I didn't catch this was merged already ;-)
@TevenLeScao do you want to handle the nit in the docs or should I do it?

Reply:
Yep sorry, maybe I merged a bit fast! If you've got time go ahead by all means, I'm on another branch right now and it's not quite yet commitable :)

Reply:
Will do when I've caught up with everything.